Makefile.PL

```diff
@@ -1,75 +1,67 @@
-
-# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v5.031.
+use 5.008005;
 use strict;
 use warnings;
-
-use 5.008005;
-
 use ExtUtils::MakeMaker;
 
 
-
 my %WriteMakefileArgs = (
-  "ABSTRACT" => "A Multi-part HTML form",
-  "AUTHOR" => "Chylli L<chylli\@binary.com>",
-  "CONFIGURE_REQUIRES" => {
-    "ExtUtils::MakeMaker" => 0
-  },
-  "DISTNAME" => "HTML-FormBuilder",
-  "EXE_FILES" => [],
-  "LICENSE" => "apache",
-  "MIN_PERL_VERSION" => "5.008005",
-  "NAME" => "HTML::FormBuilder",
-  "PREREQ_PM" => {
+    NAME                  => "HTML::FormBuilder",
+    VERSION_FROM          => "lib/HTML/FormBuilder.pm",
+    ABSTRACT_FROM         => "lib/HTML/FormBuilder.pm",
+    AUTHOR                => "Chylli L<chylli\@binary.com>",
+    CONFIGURE_REQUIRES    => {
+        "ExtUtils::MakeMaker" => 0
+    },
+    DISTNAME => "HTML-FormBuilder",
+    EXE_FILES => [],
+    LICENSE => "apache",
+    MIN_PERL_VERSION => "5.008005",
+    PREREQ_PM => {
+        "Carp" => 0,
+        "Class::Std::Utils" => 0,
+        "Encode" => 0,
+        "HTML::Entities" => 0,
+        "Moo" => 0,
+        "Scalar::Util" => 0,
+        "URI::Escape" => 0,
+        "namespace::clean" => 0
+    },
+    TEST_REQUIRES => {
+        "Test::Exception" => 0,
+        "Test::FailWarnings" => 0,
+        "Test::More" => "0.96",
+        "Test::Perl::Critic" => 0
+    },
+    test => {
+        "TESTS" => "t/*.t t/formbuilder/*.t"
+    }
+);
+
+
+my %FallbackPrereqs = (
     "Carp" => 0,
     "Class::Std::Utils" => 0,
     "Encode" => 0,
+    "ExtUtils::MakeMaker" => 0,
     "HTML::Entities" => 0,
     "Moo" => 0,
     "Scalar::Util" => 0,
-    "URI::Escape" => 0,
-    "namespace::clean" => 0
-  },
-  "TEST_REQUIRES" => {
     "Test::Exception" => 0,
     "Test::FailWarnings" => 0,
     "Test::More" => "0.96",
-    "Test::Perl::Critic" => 0
-  },
-  "VERSION" => "0.05",
-  "test" => {
-    "TESTS" => "t/*.t t/formbuilder/*.t"
-  }
-);
-
-
-my %FallbackPrereqs = (
-  "Carp" => 0,
-  "Class::Std::Utils" => 0,
-  "Encode" => 0,
-  "ExtUtils::MakeMaker" => 0,
-  "HTML::Entities" => 0,
-  "Moo" => 0,
-  "Scalar::Util" => 0,
-  "Test::Exception" => 0,
-  "Test::FailWarnings" => 0,
-  "Test::More" => "0.96",
-  "Test::Perl::Critic" => 0,
-  "URI::Escape" => 0,
-  "namespace::clean" => 0
+    "Test::Perl::Critic" => 0,
+    "URI::Escape" => 0,
+    "namespace::clean" => 0
 );
 
 
 unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
```

Comment:
wouldn't it be simpler just to require that version or perhaps 6.64 and omit all this version trouble?